clip_retrieval/clip_inference/worker.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
Can you also document it in readme so people can use it independently if they want